src/GHC.hs

```diff
@@ -1,14 +1,14 @@
 {-# OPTIONS_GHC -fno-warn-missing-signatures #-}
 module GHC (
     -- * GHC packages
-    array, base, binary, bytestring, cabal, checkPpr, compareSizes, compiler,
-    containers, deepseq, deriveConstants, directory, filepath, genapply,
-    genprimopcode, ghc, ghcBoot, ghcBootTh, ghcCabal, ghcCompact, ghci, ghcPkg,
-    ghcPrim, ghcTags, ghcSplit, haddock, haskeline, hsc2hs, hp2ps, hpc, hpcBin,
-    integerGmp, integerSimple, iservBin, libffi, mtl, parsec, parallel, pretty,
-    primitive, process, rts, runGhc, stm, templateHaskell, terminfo, text, time,
-    touchy, transformers, unlit, unix, win32, xhtml, ghcPackages, isGhcPackage,
-    defaultPackages, testsuitePackages,
+    array, base, binary, bytestring, cabal, checkApiAnnotations, checkPpr, 
+    compareSizes, compiler, containers, deepseq, deriveConstants, directory,
+    filepath, genapply, genprimopcode, ghc, ghcBoot, ghcBootTh, ghcCabal,
+    ghcCompact, ghci, ghcPkg, ghcPrim, ghcTags, ghcSplit, haddock, haskeline,
+    hsc2hs, hp2ps, hpc, hpcBin, integerGmp, integerSimple, iservBin, libffi, 
+    mtl, parsec, parallel, pretty, primitive, process, rts, runGhc, stm,
+    templateHaskell, terminfo, text, time, touchy, transformers, unlit, unix,
+    win32, xhtml, ghcPackages, isGhcPackage, defaultPackages, testsuitePackages,
```

Comment:
nit: this diff looks unnecessarily big

Reply:
@izgzhen At the moment this is unavoidable. We might refactor this to one package per line at some point if such large diffs look distracting, but it would be better to do it in a separate PR.